docs/snippets/ov_properties_api.py

```diff
@@ -1,67 +1,55 @@
-#include <openvino/runtime/core.hpp>
-
-int main() {
-//! [get_available_devices]
-ov::Core core;
-std::vector<std::string> available_devices = core.get_available_devices();
-//! [get_available_devices]
-
-//! [hetero_priorities]
-auto device_priorites = core.get_property("HETERO", ov::device::priorities);
-//! [hetero_priorities]
```

Comment:
looks like this snippet is missed

Reply:
Oh ok, I thought it is only for C++ cause on current documentation there is no tab for python

Reply:
@ilya-lavrenov in current implementation it is implemented as `@snippet snippets/ov_properties_api.cpp hetero_priorities`, which means it was not intendent to include tab for python part. Do you still want me to add tab for python example?

Reply:
yes, looks like I just missed a tab